tests/test_parser.py

```diff
@@ -3,5 +3,5 @@
 from anova_wifi import AnovaApi
 
 
-def test_can_create() -> None:
+async def test_can_create() -> None:
```

Comment:
```suggestion
@pytest.mark.asyncio
async def test_can_create() -> None:
```

Reply:
Oh yeah, my bad.

Reply:
I'm not sure if the asyncio mark decorator is still needed. I believe purest asyncio automatically sets asyncio_mode= auto and therefore the decorator is not needed. I could be mistaken. If it is still not working please put forth another PR